emhttp/plugins/dynamix/include/.login.php

```diff
@@ -193,7 +193,10 @@ function verifyUsernamePassword(string $username, string $password): bool
         appendToFile($failFile, $time . "\n");
         // Log error to syslog
         $coolReset = "";
-        if ($failCount > $maxFails) {
+        if ($failCount == ($maxFails - 1)) {
+            $error .= '<br>'.sprintf(_('Logins prevented for %s'),'<span id="countdown"></span>');
```

Comment:
Ideally we don't use `<br>` tags to create spacing.

The proper way would be to use a new `<p>` tag and the `$error` var in PHP would be an array called `$errors`. That way each one would be output in a `<p>` or `<li>`.

Granted I realize that would mean further refactors and IMO there's bigger fish to fry. So I'm not asking for any changes.

Just know in the future this might be something I push back on. :)